ml-testing-toolkit/values.yaml

```diff
@@ -109,147 +109,6 @@ connection-manager-frontend:
   #       port: 5060
   #       paths: ['/']
 
-mysql:
-  ## repo and documentation https://github.com/helm/charts/tree/master/stable/percona-xtradb-cluster
-
-  ## disable when connection-manager-backend is disabled
-  enabled: false
-  # # For Hosted Mode
-  # enabled: true
-  image:
-    repository: "percona/percona-xtradb-cluster"
-    tag: "5.7.19"
-    pullPolicy: IfNotPresent
-
-  nameOverride: mysql
-
-  replicas: 1
-
-  mysqlUser: mcm
```

Comment:
Take note of this.

So as per the above comment, we will def need a new schema added to the MySQL instance specifically for TTK.

What I am not sure about is why its called "MCM" here.

@vijayg10, can you clarify if this must be an "MCM" database, or can it be a fresh schema specifically for the TTK?

Reply:
As discussed, revert config as this will be done/tested as part of https://github.com/mojaloop/project/issues/3091

Reply:
reverted